source/particle/output/vtu.cc

```diff
@@ -35,19 +36,30 @@ namespace aspect
       {}
 
       template <int dim>
+      void VTUOutput<dim>::initialize ()
+      {
+        particle_dirname = "particles/";
+        particle_subdirectory = this->get_output_directory() + particle_dirname;
```

Comment:
why make these member variables here when you had them as just local variables in the ASCIIData class? My personal preference would to not have them as member variables, but I'm mostly interested in consistency.